pages/dashboard/index.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import Head from 'next/head';
+import styles from './dashboard.module.css';
+import transactionChartData from '../../mock/transaction-graph-data.json';
+import personData from '../../mock/person.json';
+import currencyData from '../../mock/coins.json';
+import TransactionChart from '@components/transaction-chart';
+import Breadcrumbs from '@components/breadcrumbs';
+import ProfileSidebar from '@components/profile-sidebar';
+import Wallets from '@components/wallets';
+import ChartActions from '@components/chart-actions';
```

Comment:
Import order
1. external lib
2. Components
3. Redux store or related to redux if exists
4. Constants
5. Css